src/sound.h

```diff
@@ -44,6 +45,7 @@ struct SimpleSoundSpec
 	std::string name;
 	float gain;
 	float fade;
+	float pitch;
```

Comment:
same for those params, please set default values here (you cannot remove the params affectations)